client/layout/logged-out.jsx

```diff
@@ -54,6 +54,10 @@ const LayoutLoggedOut = ( {
 	const isJetpackCheckout =
 		sectionName === 'checkout' && window.location.pathname.startsWith( '/checkout/jetpack' );
 
+	const isJetpackThankYou =
```

Comment:
Is this a good practice? There doesn't seem to be another way to prevent the master bar from rendering.